src/FluentMigrator.Abstractions/Model/ColumnDataType.cs

```diff
@@ -34,5 +34,11 @@ public class ColumnDataType : IColumnDataType
 
         /// <inheritdoc />
         public string CustomType { get; set; }
+
+        /// <inheritdoc />
+        public string Expression { get; set; }
+
+        /// <inheritdoc />
+        public bool ExpressionStored { get; set; }
```

Comment:
@PhenX @Shane32 This is why the unit test is failing.  Should it default to true?

Reply:
I designed as follows: `Expression` is a string containing the computed column SQL expression if it is a computed column; otherwise `null`.  `ExpressionStored` is always `false` unless (a) it is a computed column, and (b) the column is persisted/stored (vs a virtual column).